infra/terraform/modules/aws_account_logging/vpcflowlogs/vpcflowlogs_to_elasticsearch.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+import gzip
+import logging
+import os
+from itertools import islice
+import boto3
+import certifi
+import elasticsearch.helpers
+import jsonpickle
+from elasticsearch import Elasticsearch
+from flowlogs_reader import FlowRecord
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(os.environ.get('LOG_LEVEL', 'DEBUG'))
+
+creds = os.environ['ES_USERNAME'] + ':' + os.environ['ES_PASSWORD']
```

Comment:
Mixing string concatenation and string.format - pick one (hint `.format` is better)